File: College-Sports-Domain/Models/Event.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;
namespace College_Sports_WebApp.Database.Models{ 

    public class Event
    {
        [JsonNumberHandling(JsonNumberHandling.AllowReadingFromString)]
        public int id { get; set; }
        public string uid { get; set; }
        public string date { get; set; }
        public string name { get; set; }
        public string shortName { get; set; }
        public Season season { get; set; }
        public List<Competition> competitions { get; set; }
        public List<Link> links { get; set; }
        public Status status { get; set; }
        public Weather weather { get; set; }
    }

}

File: College-Sports-WebApp/ClientApp/src/api/models/event.ts
/* tslint:disable */
/* eslint-disable */
import { Competition } from '../models/competition';
import { Link } from '../models/link';
import { Season } from '../models/season';
import { Status } from '../models/status';
import { Weather } from '../models/weather';
export interface Event {
  competitions?: Array<Competition> | null;
  date?: string | null;
  id?: number;
  links?: Array<Link> | null;
  name?: string | null;
  season?: Season;
  shortName?: string | null;
  status?: Status;
  uid?: string | null;
  weather?: Weather;
}
